components/camera/fake/camera.go

```diff
@@ -433,3 +422,10 @@ func (c *Camera) Close(ctx context.Context) error {
wn3Hod8OHrPL1q7iH564L4f2HiTM91N6uG8f4Vu22lDvYLNrml6iTf8i+r50jt+FvtXeWL0nJ82fPt75mfS++EWMnzZ873sWazPTKnnjJ5vdUz1e5jax31+XpM5LFzNOSiRFO1MdeN6slyH2t11F091RLT5jXvupLFNVXTTO1ltVZZiuPmOWS9sSQz1F0b6chFBq59Js0ipH4+LLjQjWzRdV5odGXJEM2uRScbNfLiQyJw3ytnZVssvK/2lU+C6WLFc96V9tRq1s7W/uRG2q18i9VrH4sndVS6lcvNzEklarBYnHZZGu3qbNZbtZM82zXvXqZfozOOMm201sixW6SNqZM7WueZOtcdmy2tdFdXNld6aUiVJelpm1YrXmeRs6I687Y5Gv4ndfb7gTcqpnka1rZ5vtO2tn8itsiTaRfM4AcFAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAdVnVprDI4BQ/Rsu4pxzgHDOI16vX2USk+JSx0VLaseedcs8ln2HhN5XmjNfwn4j/und/f8Lktql2N/Xiq3/wAGbzaf3L1z7XqNbi+1rSW6r25nsj/cdeOXEseUnrkzDb95cjPbymTMFvFiRTtTHRNa9VyclvUtpVcil1yZKjyztqxMWjfpekuioks2a7s2yevKpY263qiE8vT0lOerT8iEj9jfQ7HParxNj7ihpAn0MhXcUJSvOqt7GNNiCWr6ZLZVfTq+H8xvpmOqTsR3M3dxwi1batvuIdxXtjw/6bFU/Cd9BS17x9Nem1tS8xPpuNe0jon+ZZHKlF7Mlr6UiqMX60NZRVomYI3xZGua5HdQqzGu2btzK8i1sr1cgxc6rT4mvdNcydpGVOzZg7kn7kruijwxZWQsc5qpEAHNoAAAAAAAAAAAAAAAAAAAAAAACWRw6cDHAAGgAAAAAAAAAA9d/DjvAu7veLbTyP8Ay89bbbcV9rRTZYP5WVbfofUO8mw9DdSKqw1YNPNOrxq180fAj7x3Y4tXvZ3Xivd6t/w1Lbbr4pI0vwZ34uq02yWSdTv09nOf683/AKK43/HkOJQaWzzu7hPbcU22Ct2HmN5Dk2VbYTeXnrpZQZLdw5ZmOfMx1cJO2qviROATrbSzsltbzIs4aBw6nkAOE6kSSNal41tpJ/dbmvT6l9PzK+kjqAlaW75tfyRDMM4gxPV7HaPPpLYdnaTqLFt/St4kihxtFVueRkHHqqattvbV7GtcddNEVM2padCNVksRzxLEqlbOphqVkjXZdI8Cg5zbQACGgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAOnpe5Hee3dfi1NxZu21mXobuNfVBfnZfmo0rVPNAVxKrLqrq6utffuN8PqkrxtSQz0UsV1ytSyzTX6Hh97t8s1kb38OO9dd5Au7u/ky9+HzX5Vtjq29reyvg6N4K2a9zKcZ4TaGS+HJ4rsPdxONY8PMJXtPn+8h5mGnhyxR7De7Pngef3m3abOL0Qmwx1Yk5I9LIoLROltosMyBogCQyAiCzSR0sCIO5NFkNU3iBH0b2NvabCSR5vkS5G1Dfo83m6vNpA6obQdNukq3E3rSuxsz7qSVfjrO1enP6tPw42NS9Pq05JgK30/3HDiqLPIpFuM05q6bfM2rXyNWaRWaJXFUF2nCNrexzlbUbWzIgHO1UAAwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABOKW8N63pZ1tV5prBpo+0d1O9MHfHZR8P3t1Xi8FNNZLNJbuOvvZ8vVr7/EfFC7bbibaTUmhvaOSOytW9Xk00dOnslG9pz7uup1lvrfFOGXhtatqtNHl97s888D2Xdrvjte+MVdlxK1dvxalco5eUe7y/srP/W+k1uLcItDeydcmv7T0X/Tyx9dcrq6fON1tWszViqk9LPUbzZ5+2JhtxtXR5pZHN6dVenXI0ZaOlmjbrLlgyqbrKtlNYI66tDT4mLSAOAGszsfTYHaWdGmuaea+aA2QUOSxB3szU43pN07R0pd1yrn1fVbP2ZV6tbfUjWImDd1ld75ldb+zIyW6unkPSvJLcoOtN4srtY5yvFUWsQBw5a0AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEq3tVpp5Ne6wZ9K7s/xGhmhrw/vDW8qS0xcQXVLXsrOsutfn8yPmYLh2XC9pE+up1lvtnEOCV0rc7Vxbjb3Wqk0WKaZ5XiGw5tI8zwHvVxTu9Jq2kuqJ/4m2l64L+FqPDHwxPomx7zd3e8qVLyV4XvLLyT/wCA3+Sf27Os7evThKv1f9fPt3s7VbdUaD9SPmfTeJd25Yqu2hXq+V6ZNP8AkeU3nCMX05ErjPfw89rrZY4Mjgbe42F4/Y1vSKdFYJ2o0RyMHdXghqXYcOAWYEdCzwGbLKV9wK7UZUbN+Rqt4mXId1DMg5EVO7ZzuSqpO8meCwKjoJu9bjgAMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGe4L3t4zwTKu13LcPvtpfxIWv3Lcv0aPVbfv5wffrRxPht9vf3n2l9VG/8ARv8A9x83zyO5s6Rk53Cn1T0OA8UTey4rs82s1FuLehJl4qXSll8zS3nc3drqjprzxzj6k125rBnzjN9psQb7dbR6tvPNDb3cclq55duTWfP3FSKgz+64FvIbNWjay8DHybKWuFomWRd7+PR/+4T3/wBS3qf18yz/AMY8ZfOaK/7+227/APTJ1rS9Nr6TnpvsNiTvPxKTneH9NtB/2GrfjHEJf+PZfuVrT+qkNE/Rl+BkbyWph0rw1Y/2GpLNNI85JL3f5rWf7SoapdaZsrd2yIM1rgAMAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB//2Q==")
```

Comment:
I decided to add back these module constants & remove the files on disk as I can't embed a fake camera in a module without it being in the RDK repo if it depends on external files. That dramatically reduces the utility of fake camera.